content/tools/other/rdp-sec-check/index.md

```diff
@@ -16,6 +16,11 @@ Is a Perl script to enumerate security settings of an RDP Service (AKA Terminal
 ## Installation
 
 ```plain
+sudo cpan
```

Comment:
Preferably not install with elevated privileges when not needed. Can you fix this and merge into one line? `cpan install Encoding::BER`

Reply:
Verified and updated installation steps.

Reply:
Thanks!